src/DotNet/DotNet.csproj

```diff
@@ -171,6 +174,32 @@
         VersionBand="$(DotNetPreviewVersionBand)"
     />
 
+    <!-- HACK: temporary fixes for iOS and MacCatalyst workloads -->
+    <ReplaceText
+        Input="$(DotNetSdkManifestsDirectory)microsoft.net.sdk.maccatalyst/WorkloadManifest.json"
+        Output="$(DotNetSdkManifestsDirectory)microsoft.net.sdk.maccatalyst/WorkloadManifest.json"
+        OldValue="&quot;description&quot;"
+        NewValue="&quot;extends&quot;: [ &quot;microsoft-net-runtime-maccatalyst&quot; ], &quot;description&quot;"
+    />
+    <ReplaceText
+        Input="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        Output="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        OldValue="Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst&quot;"
+        NewValue="Condition=&quot;%24([MSBuild]::IsOSPlatform('osx'))&quot; Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst&quot;"
+    />
+    <ReplaceText
+        Input="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        Output="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        OldValue="Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm&quot;"
+        NewValue="Condition=&quot;%24([MSBuild]::IsOSPlatform('osx'))&quot; Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm&quot;"
+    />
+    <ReplaceText
+        Input="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        Output="$(DotNetSdkManifestsDirectory)microsoft.net.workload.mono.toolchain/WorkloadManifest.targets"
+        OldValue="Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64&quot;"
+        NewValue="Condition=&quot;%24([MSBuild]::IsOSPlatform('osx'))&quot; Sdk=&quot;Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64&quot;"
+    />
```

Comment:
/cc @Redth @steveisok 

I had to manually patch up `microsoft.net.workload.mono.toolchain/WorkloadManifest.targets` for iOS and MacCatalyst projects to be able to build on Windows.

These AOT packs have `platforms` limiting to osx:

https://github.com/dotnet/runtime/blob/685eda620e0f058b157b6d0a66da6f21c2d165fa/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in#L55

And so on Windows, imports such as these would fail:

https://github.com/dotnet/runtime/blob/685eda620e0f058b157b6d0a66da6f21c2d165fa/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets#L23-L25

Not sure if we should merge without this fixed, as it would probably break anyone doing `maui-check --main`.

Reply:
Created an issue on the runtime end and we'll see what we can do.

Reply:
If you try to build an iOS or Mac Catalyst project on Windows right now, you'll hit:
```
C:\src\maui\bin\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.targets(23,9): error : C:\src\maui\bin\dotnet\sdk\6.0.100-preview.6.21328.1\Sdks\Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm\Sdk not found. Check that a recent enough .NET SDK is installed and/or increase the version specified in global.json.
C:\src\maui\bin\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.targets(24,9): error : C:\src\maui\bin\dotnet\sdk\6.0.100-preview.6.21328.1\Sdks\Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64\Sdk not found. Check that a recent enough .NET SDK is installed and/or increase the version specified in global.json.
C:\src\maui\bin\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.targets(29,9): error : C:\src\maui\bin\dotnet\sdk\6.0.100-preview.6.21328.1\Sdks\Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst\Sdk not found. Check that a recent enough .NET SDK is installed and/or increase the version specified in global.json.
```